pkg/config/kube/config.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2022 flannel authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kube
+
+import "github.com/flannel-io/flannel/pkg/ip"
+
+type KubeSubnetMngrConfig struct {
+	Networks     []ip.IP4Net
+	IPv6Networks []ip.IP6Net
```

Comment:
I guess you are preparing for the new kube API, right? I wonder if we should have both "Network" and "Networks" or just have Networks and check in etcd that it only has one element